app/assets/javascripts/calendars/guider-slot-picker.es6

```diff
@@ -122,20 +184,23 @@
       const events = $(event.currentTarget).data('events'),
       calendarView = this.$el.fullCalendar('getView'),
       calendarStartDate = calendarView.intervalStart,
-      calendarEndDate = calendarView.intervalEnd;
+      calendarEndDate = calendarView.intervalEnd.subtract(2, 'days'),
+      eventsToAdd = [];
 
       for (let currentDate = moment(calendarStartDate); currentDate < calendarEndDate; currentDate.add(1, 'days')) {
         for (let eventIndex in events) {
           event = events[eventIndex];
```

Comment:
Have I misread this or are we overwriting the argument to the function here? We might have got round a missing lint warning here.